modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/message/MessageServiceImpl.java

```diff
@@ -121,6 +122,10 @@ public void register(MessageListener lsnr, short type) {
     }
 
     private void onMessage(String consistentId, NetworkMessage msg) {
+        if (msg instanceof CancelOperationRequest) {
```

Comment:
cannot use multiple message groups in a single module https://issues.apache.org/jira/browse/IGNITE-23798